serde_codegen/src/lib.rs

```diff
@@ -32,6 +37,13 @@ include!(concat!(env!("OUT_DIR"), "/lib.rs"));
 include!("lib.rs.in");
 
 #[cfg(feature = "with-syntex")]
+pub fn expand(src: &Path, dst: &Path) -> io::Result<()> {
```

Comment:
Good idea, although [Syntex::expand accepts &Path](https://github.com/serde-rs/syntex/blob/bf7b812bedab8f1f02ad7e0cd237c48f30dd9e1d/syntex/src/lib.rs#L126). I made the change here and filed https://github.com/serde-rs/syntex/issues/64 to change it in Syntex.